docs/module/srv6.md

```diff
@@ -28,26 +71,23 @@ We want to use unnumbered P2P interfaces and dual-stack deployment with IPv6 con
 ```
 addressing:
   p2p:
-    unnumbered: true
+    ipv6: true
```

Comment:
This will result in numbered IPv4 interfaces. Is that what you want? What's the point of SRv6 in this example?

Reply:
Changed the example to a simplified version of test case 01